### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by chensteven** [yjn870/SRCNN-pytorch#4]
Hi, 
1. Why did you do zero-padding?
2. Why use Adam instead of the SGD? How come you didn't just use SG like the paper?
3. Why remove the weights initialization? Why remove it?

**Comment by chensteven** [comment]
Also, how come you don't crop patches for the training dataset? Thanks!

**Comment by dxli94** [comment]
It is not an obligation for authors to contribute to the open-source community. 

People should learn to appreciate others' great efforts and be polite and gentle.
